### PR TITLE
New error handling guide page

### PR DESCRIPTION
**Pull request by baronkobama**
This PR adds a new page to the Popular Topics section of the guide named Error Handling. This page goes into detail about all 4 various methods of error handling (local try/except, per-command, per-cog, and global) and provides examples of each.

### CODE REVIEW

File: docs/popular-topics/error-handling.mdx [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+
+For these types of commands, it is recommended to utilize global error handling 
+until [the feature](https://github.com/Pycord-Development/pycord/issues/1388) is added.
+:::
+
+## Basic Handling
+
+The most basic way to handle errors is to tackle it at the source. This can be done like so:
+
+:::tip Multiple Errors
```
Comment (EmreTech) [inline docs/popular-topics/error-handling.mdx:33]: This tip feels a little misplaced

File: docs/popular-topics/error-handling.mdx [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+
+```python title="./main.py"
+# This code requires the `message_content` privileged intent for prefixed commands.
+
+import asyncio
+
+import discord
+
+intents = discord.Intents.default()
+intents.message_content = True  # To use this, the message content privileged intent must be on in the dev portal.
```
Comment (EmreTech) [inline docs/popular-topics/error-handling.mdx:47]: The comment looks redundant

File: docs/popular-topics/error-handling.mdx [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+
+bot = discord.Bot(debug_guilds=[...], intents=intents)
+
+
+@bot.slash_command(description="Gets some feedback.")
+@discord.option("name", description="Enter your name.")
+async def feedback(ctx: discord.ApplicationContext, name: str):
+    try:
+        await ctx.respond(f"Hey, {name}! Send a message within the next 30 seconds please!")
```
Comment (EmreTech) [inline docs/popular-topics/error-handling.mdx:56]: Maybe rephrase this message? It sounds like it's missing context for some reason

File: docs/popular-topics/error-handling.mdx [hunk 4]
```diff
@@ -0,0 +1,356 @@
...
+@bot.slash_command(description="A private command...")
+@commands.is_owner()  # This decorator will raise commands.NotOwner if the invoking user doesn't have the owner_id
+async def secret(ctx: discord.ApplicationContext):
+    await ctx.respond(f"Hey {ctx.author.name}! This is a secret command!")
+
+
+@secret.error
+async def on_application_command_error(ctx: discord.ApplicationContext, error: discord.DiscordException):
+    if isinstance(error, commands.NotOwner):
+        await ctx.respond("Sorry, you can't use this command!")
```
Comment (EmreTech) [inline docs/popular-topics/error-handling.mdx:142]: Maybe rephrase to `"You're missing permissions to use this command!"`

**Review by EmreTech — COMMENTED** [review]
Overall pretty good, here are some changes I suggest

File: docs/popular-topics/error-handling.mdx (hunk 1 above)
Comment (baronkobama) [reply docs/popular-topics/error-handling.mdx:33]: Felt so too, will probably move that as well as the mixing handlers tip into a small FAQ section.

File: docs/popular-topics/error-handling.mdx (hunk 3 above)
Comment (baronkobama) [reply docs/popular-topics/error-handling.mdx:56]: Had actually already rephrased it slightly below in the message preview and just forgot to do it in the code example lmao

Comment (EmreTech) [reply docs/popular-topics/error-handling.mdx:56]: Oh lol

**Review by EmreTech — APPROVED** [review]
LGTM

**Comment by baronkobama** [comment]
Made some last-minute changes, apologies. Also, the guide will not compile currently due to the changes merged in ce1d57.
